lib/arjdbc/jdbc/arel_support.rb

```diff
@@ -1,3 +1,31 @@
+if Arel::VERSION >= "7.1.0"
```

Comment:
not sure about the right place to do the patch - maybe place these where visitors live (also make sure arel is loaded).
new features are usually adapter to work backwards although here (quick look a arel's commits) the patch might do.